realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -76,6 +76,7 @@ protected BaseRealm(RealmConfiguration configuration, boolean autoRefresh) {
         this.configuration = configuration;
         this.sharedGroupManager = new SharedGroupManager(configuration);
         this.schema = new RealmSchema(this, sharedGroupManager.getTransaction());
+        this.handlerController = new HandlerController(this);
```

Comment:
None, but it possible for a Looper thread to enable/disable autorefresh without loosing all registered listeners. We had a feature request for it #2051